ui/build.gradle

```diff
@@ -0,0 +1,33 @@
+plugins {
+    id 'com.github.node-gradle.node' version '3.1.0'
+}
+
+node {
+    download = true
+    version = "12.16.2"
+}
+
+task check(type: YarnTask) {
+    args = ['unit-coverage']
+
+    inputs.files('package.json', 'package-lock.json', 'tsconfig.json')
```

Comment:
yarn uses a `yarn.lock` file. Did you mean to use that instead of `package-lock.json`